spec/fabricators/files/no_keywords.csv

```diff
@@ -0,0 +1 @@
+Football Player
```

Comment:
The file name is `no_keywords.csv` but here is the `Football Player` content 🤔

Reply:
It is a CSV header. Should I use csv with header or without header?

Reply:
`Football Player` doesn't seem like the header 🤔

The header should be something generic like `Title/Name/Keyword`, what do you think?

IMO I think we can go without header in the CSV file, but it's up to you, just making sure that we go with the `no confuse header title` :D

Reply:
Agree with you

Reply:
Fixed in a2a45ce95021f622230bdcb9291fdb7f095a2316